src/target/riscv/riscv-013.c

```diff
@@ -829,38 +902,35 @@ uint32_t riscv013_access_register_command(struct target *target, uint32_t number
 	return command;
 }
 
+static bool is_command_unsupported(struct target *target, uint32_t command)
+{
+	bool supported = !ac_cache_contains(&get_info(target)->ac_not_supported_cache, command);
+	if (supported)
+		return false;
+
+	LOG_TARGET_DEBUG(target, "Abstract command 0x%"
+			PRIx32 " is cached as not supported", command);
+	log_debug_reg(target, get_cmdtype(command),
+			command, __FILE__, __LINE__, __func__);
+	return supported;
```

Comment:
1) There seems to be a bug - this function always returns `false`.

2) Please, don't name the variable `supported`. The fact that the command is not in the cache does not mean that it is supported. You can call it `already_seen_unsupported`, `is_unsupported`, `known_unsupported` or similar.

```suggestion
	bool already_seen_unsupported = ac_cache_contains(&get_info(target)->ac_not_supported_cache, command);
	if (!already_seen_unsupported)
		return false;

	LOG_TARGET_DEBUG(target, "Abstract command 0x%"
			PRIx32 " is cached as not supported", command);
	log_debug_reg(target, get_cmdtype(command),
			command, __FILE__, __LINE__, __func__);
	return true;
```

Reply:
Thanks! Addressed